pytest_dcos/plugin.py

```diff
@@ -6,6 +6,29 @@
 
 logger.setup(os.getenv('LOG_LEVEL', 'DEBUG'))
 
+log = logging.getLogger(__name__)
+
+USER_HOME_DIR = os.path.join(os.path.expanduser('~'))
+
+
+def order_fixtures(metafunc):
```

Comment:
I'm fine with this but we need to remove https://github.com/dcos/dcos/blob/master/packages/dcos-integration-test/extra/conftest.py#L22
and 
https://github.com/dcos/dcos/blob/master/packages/dcos-integration-test/extra/conftest.py#L17

and also refactor the one place that is used, which is here:
https://github.com/dcos/dcos/blob/master/packages/dcos-integration-test/extra/test_composition.py#L19